e2e-tests/demand-backup/run

```diff
@@ -101,7 +101,7 @@ function main {
 	kubectl_bin delete perconapgcluster/${cluster}
 	sleep 30
 
-	gsutil rm -r gs://${BUCKET}/backrestrepo/${cluster}-backrest-shared-repo
+	gsutil -m rm -r gs://${BUCKET}/backrestrepo/${cluster}-backrest-shared-repo
```

Comment:
-m option could lead to gsutil infinite waiting time.

Reply:
I have tested it and it works. And we save +- 5-7 mins